scripts/mkcloud

```diff
@@ -353,10 +353,15 @@ function onhost_create_cloud_lvm()
         _lvcreate $cloud.node$i $hdd_size $cloudvg $next_pv_device
     done
     if [ $controller_raid_volumes -gt 1 ] ; then
+        # total wipeout of the disks used for RAID, to prevent bsc#966685
+        volume="/dev/$cloudvg/$cloud.node1"
+        dd if=/dev/zero of=$volume bs=1G count=${controller_hdd_size}
```

Comment:
I mean, there doesn't need to be count argument, but than the command ends ugly with error (no space left). I though this is nicer, esp. when the provided size is alwyas in GB's.